src/ngraph/serializer.cpp

```diff
@@ -2894,6 +2901,13 @@ json JSONSerializer::serialize_node(const Node& n)
         node["initial_axes"] = serialize_axis_set(tmp->get_initial_broadcast_axes());
         break;
     }
+    case OP_TYPEID::Bucketize_v3:
+    {
+        auto tmp = static_cast<const op::v3::Bucketize*>(&n);
+        node["output_type"] = write_element_type(tmp->get_output_type());
+        node["with_right_bound"] = tmp->get_with_right_bound();
+        break;
+    }
```

Comment:
Don't need this since the op has attribute_visitor.

Reply:
No changes should be in serializer.